integrations/oci/oci/src/main/java/io/helidon/integrations/oci/OciConfigProvider.java

```diff
@@ -71,7 +73,14 @@ private static void create() {
                 ConfigSources.systemProperties(),
                 ConfigSources.file("oci-config.yaml").optional(true),
                 ConfigSources.classpath("oci-config.yaml").optional(true));
-
-        ociConfig = OciConfig.create(config);
+        if (config.get(CONFIG_PREFIX).exists()) {
+            ociConfig = OciConfig.create(config.get(CONFIG_PREFIX));
+        } else {
+            ociConfig = OciConfig.create(config);
```

Comment:
I realized we cannot do this - as the configuration is not required (there are reasonable defaults), you would always warn, unless somebody adds the `oci-config.yaml` and adds at least one `helidon.oci` property there.
So either we warn if we find one of the supported options (I think this is too complicated to implement maybe?), or just log it in `TRACE`.

Reply:
Yeah, 1st option is complicated, so I adopted the 2nd option to just log it in `TRACE`